SPECS/mariner-repos/mariner-official-base.repo

```diff
@@ -7,3 +7,23 @@ repo_gpgcheck=1
 enabled=1
 skip_if_unavailable=True
 sslverify=1
+
+[mariner-official-base-source]
+name=CBL-Mariner Official Base Sources
+baseurl=https://packages.microsoft.com/cbl-mariner/$releasever/prod/base/srpms/
+gpgkey=file:///etc/pki/rpm-gpg/MICROSOFT-RPM-GPG-KEY file:///etc/pki/rpm-gpg/MICROSOFT-METADATA-GPG-KEY
+gpgcheck=1
+repo_gpgcheck=1
+enabled=0
+skip_if_unavailable=True
+sslverify=1
+
+[mariner-official-base-debuginfo]
```

Comment:
Is this part needed, since it's already available in mariner-debuginfo.repo?

Reply:
It matches what other RPM based distributions do, which means:
- `dnf debuginfo-install` and `dnf download --debuginfo` work like other RPM based distributions (they enable the debuginfo repos)
- articles like https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/developer_guide/intro.debuginfo work for mariner too